UnityProject/Assets/Scripts/Weapons/Projectiles/BulletBehaviour.cs

```diff
@@ -71,6 +75,20 @@ public virtual void Shoot(Vector2 dir, GameObject controlledByPlayer, Gun fromWe
 	{
 		isSuicide = false;
 		StartShoot(dir, controlledByPlayer, fromWeapon, targetZone);
+		if (isRangeLimit);
+		{
+			StartCoroutine(countTiles());
+		}
+		
+	}
+
+	private IEnumerator countTiles()
+	{
+		Vector2 startPos = gameObject.AssumedWorldPosServer();
```

Comment:
so why do you need `startPos`? it's not used

Reply:
this is reusing code from the pka's bullet

Reply:
then it's worth removing both from bulletbehaviour and kineticbullet